src/core/thread/mesh_forwarder.cpp

```diff
@@ -184,32 +184,6 @@ void MeshForwarder::PrepareEmptyFrame(Mac::TxFrame &aFrame, const Mac::Address &
     aFrame.SetPayloadLength(0);
 }
 
-void MeshForwarder::EvictMessage(Message &aMessage)
-{
-    PriorityQueue *queue = aMessage.GetPriorityQueue();
-
-    OT_ASSERT(queue != nullptr);
-
-    LogMessage(kMessageEvict, aMessage, kErrorNoBufs);
-
-    if (queue == &mSendQueue)
```

Comment:
Adding a note since this method is now removed:

The `EvitMessage(Message &aMessage)` method could be used even when the `Message` was not from the `MeshForwarder`'s `mSendQueue` (as it checked the queue). However, it was only used from the `MeshForwarder` (it is `private`) with messages from `mSendQueue`.